### PR TITLE
Update OCP daily SQL to merge node labels with pod and volume labels.

### PR DESCRIPTION
**Pull request by chambridge**
* Update daily SQL
* Add data generation and test entries

### CODE REVIEW

File: koku/masu/database/sql/reporting_ocpstoragelineitem_daily.sql [hunk 1]
```diff
@@ -40,6 +40,9 @@ CREATE TEMPORARY TABLE reporting_ocpstoragelineitem_daily_{{uuid | sqlsafe}} AS
         ON rp.provider_id = p.uuid
     LEFT JOIN volume_nodes_{{uuid | sqlsafe}} as uli
         ON li.id = uli.id
+    LEFT JOIN {{schema | sqlsafe}}.reporting_ocpnodelabellineitem_daily as nlid
+        ON uli.node = nlid.node
```
Comment (adberglund) [inline koku/masu/database/sql/reporting_ocpstoragelineitem_daily.sql]: We also need to join on date here.

File: koku/masu/database/sql/reporting_ocpstoragelineitem_daily.sql [hunk 2]
```diff
@@ -52,7 +55,8 @@ CREATE TEMPORARY TABLE reporting_ocpstoragelineitem_daily_{{uuid | sqlsafe}} AS
         li.persistentvolume,
         li.storageclass,
         li.persistentvolume_labels,
-        li.persistentvolumeclaim_labels
+        li.persistentvolumeclaim_labels,
+        nlid.node_labels
```
Comment (adberglund) [inline koku/masu/database/sql/reporting_ocpstoragelineitem_daily.sql]: I'm thinking about this, as to whether there's a good way to do this without grouping by the node labels. Right now if a record from the line item table happens to match multiple rows in the node label table then we'll likely duplicate some usage data.

Comment (chambridge) [reply koku/masu/database/sql/reporting_ocpstoragelineitem_daily.sql]: Updates applied

File: koku/masu/database/sql/reporting_ocpstoragelineitem_daily.sql (hunk 1 above)
Comment (chambridge) [reply koku/masu/database/sql/reporting_ocpstoragelineitem_daily.sql]: Updates applied.

**Comment by chambridge** [comment]
I've got some weird codecov thing going on ...

**Pull request merged**